xlators/mgmt/glusterd/src/glusterd-handler.c

```diff
@@ -5841,8 +5833,6 @@ glusterd_get_state(rpcsvc_request_t *req, dict_t *dict)
 
         fprintf(fp, "Volume%d.snap_count: %" PRIu64 "\n", count,
                 volinfo->snap_count);
-        fprintf(fp, "Volume%d.stripe_count: %d\n", count,
-                volinfo->stripe_count);
```

Comment:
this removal may result in checksum mismatch during update... can you check if we need to write it regardless.

@mohit84 please confirm.